src/main/java/com/zombiecastlerush/building/Item.java

```diff
@@ -0,0 +1,35 @@
+package com.zombiecastlerush.building;
+
+public class Item {
+    private String name, description;
+
+
+    public Item(String name, String description){
+        setName(name);
+        setDescription(description);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    @Override
+    public String toString() {
+        return "Item{" +
+                "name='" + name + '\'' +
```

Comment:
Is this supposed to be a line break, where you have the '\'

Reply:
{"name": "myName","description": "myDescription"}
this can be simply used as a Json string.
Or you can call ItemFactory.itemToJson(this). this function converts a Item object to Json string unless you want to customize your toString()